docs/wiki/user/ts-channel-switching.md

```diff
@@ -4,10 +4,11 @@ title: TeamSpeak Channel Switching
 
 ## Setup
 
-ACRE2 includes the option to automatically switch to a dedicated TeamSpeak 3 channel upon game start. This is enabled by default and can be turned off with CBA settings. Players are moved when the game starts, after the briefing (upon pressing "Continue").
+ACRE2 includes the option to automatically switch to a dedicated TeamSpeak 3 channel upon game start. This is enabled by default and can be turned off. Players are moved when the game starts, after the briefing (upon pressing "Continue"). This feature can be disabled in the addon settings.
 
 Dedicated TS3 channels need to be created for this to function. Channel names must contain "ACRE" to be detected as a dedicated channel. 
 Channel names can contain all or part of the Arma 3 server name. If this is the case, the best matching channel name will be selected to switch to. This allows for multiple ACRE dedicated channels to be active at once for multiple Arma servers.
+It is also possible to set the name of the desired channel in the addon settings. When this is filled in, the best matching channel will be used. When empty, the Arma 3 server name will be used as above.
```

Comment:
You'll need an empty line between each paragraph, otherwise they will all jump to the same line and just wrap. You can see that on current live doc already.